docs/platforms/javascript/common/migration/v7-to-v8/index.mdx

```diff
@@ -20,7 +20,7 @@ Before updating to `8.x` of the SDK, we recommend upgrading to the latest versio
 npx @sentry/migr8@latest
 ```
 
-Our migration tool will let you select which updates to run, and automatically update your code. In some cases, we cannot automatically change code for you. These will be marked with a `TODO(sentry)` comment instead. Make sure to review all code changes after running `@sentry/migr8`! For more details on the deprecations, see our docs on [Deprecations in 7.x](./v7-deprecations).
+Our migration tool will let you select which updates to run, and automatically update your code. In some cases, we cannot automatically change code for you. These will be marked with a `TODO(sentry)` comment instead. Make sure to review all code changes after running `@sentry/migr8`! For more details on the deprecations, see our docs on [Deprecations in 7.x](./v7-deprecations). Despite having `@sentry/migr8`, we still recommend reading the migration guide, since `@sentry/migr8` does not cover all of the changes needed to migrate.
```

Comment:
```suggestion
Our migration tool will let you select which updates to run, and automatically update your code. In some cases, we cannot automatically change code for you. These will be marked with a `TODO(sentry)` comment instead. Make sure to review all code changes after running `@sentry/migr8`! 

For more details on the deprecations, see our docs on [Deprecations in 7.x](./v7-deprecations). 

<Alert level="warning">
Despite having `@sentry/migr8`, we still recommend reading this migration guide, since `@sentry/migr8` does not cover all of the changes needed to migrate.
</Alert>
```
If there's only a few additional changes needed, it would be great if we could point those out. If not, can leave this as is.